docs/readme.md

```diff
@@ -0,0 +1,103 @@
+- They have legacy AngularJS apps they need to migrate. NgUpgrade is great, but still a lot of work to set up.
+- They want to write robust, proven code: error handling, race conditions.
+
+**Nx** helps achieve those requirements.
+- It gives you Nx Workspace, a monorepo way of building frontend apps. The same approach used by Google, Facebook, Uber, Twitter and many others.
+- It gives you code generation abilities to generate state management stuff and routing. It also comes with runtime libraries to make your code more robust. It comes with a code formatter (used by Google!).
+- It comes with code generation abilities to set up a hybrid app in minutes. So you can take advantage of your existing AngularJS code right away.
+
+#### Getting Started
+Since **Nx** is designed to enhance the AngularCLI you will want to have that installed globally along with `@nrwl/schematics`:
+
+```
+yarn global add @angular/cli
+yarn global add @nrwl/schematics
+```
+
+>A note on [Yarn](https://yarnpkg.com/en/)  
+Yarn does a better job at recognizing and dealing with multiple peer dependencies on globally installed
+packages, thus is recommended (but not a hard requirement)
+
+While it is possible to leverage the **Nx** generate schematics and the libraries in a default AngularCLI application,
+**Nx** is designed to help you create and build enterprise grade Angular applications. It provides an opinionated
+approach to application project structure and patterns.
+
+That approach begins with the concept of a workspace, or what we call a **Nx Workspace**.
+
+## Nx Workspace
+This workspace contains your Angular application (or applications) and any supporting libraries you create. It is a mono
+repo for your application domain.
+
+[Why a workspace (or mono repo) is needed](why-a-workspace.md)
+
+The AngularCLI can actually be set up manually to support multiple Angular applications and referenced library
+code. It doesn't do this out of the box for you though. But **Nx** does! You can use the **Nx** `new`
+schematic to create a new **Nx Workspace**:
+
+```
+ng new myprojectname --collection=@nrwl/schematics
+```
+
+The `new` schematic for **Nx** handles doing all that heavy lifting for you. What you end up with is
+a **Nx Workspace** with the following files/folders created:
+```
+apps/
+libs/
+.angular-cli.json
+tslint.json
+test.js
+tsconfig.json
+tsconfig.app.json
+tsconfig.spec.json
+tsconfig.e2e.json
+```
+It's similar to the standard CLI projects with a few changes:
+- We have the `apps` dir where all applications are placed
+- We have the `libs` dir where all custom library code is placed
+- Some files have moved to the root: `tsconfigs, test.js`, so all apps and libs share them.
+
+All of the standard AngularCLI commands are available, again because **Nx** simply extends the power of the
+AngularCLI.
+
+You can build/serve/test any of the apps by passing the app name.
```

Comment:
I would not talk about `build` here because there is nothing in an empty workspace that can be built.

Instead, I'd talk about generating apps and libs here. How to generate an app, how to generate a lib. What is an app? What is a lib? How an app can depend on a lib. 

Something like that:

```
## Apps and Libs

* App is something we actually ship to the user (it is a bundle or a group of bundles). It's akin to a binary.
* Libs is something we compose apps from. We don't ship libs to the user.

This is how you generate app: ... I would merely mention schematics here with a link to a maybe footnote.
This is how you generate lib: ...
This is how your app or or your lib can depend on another lib (mention npmscope)

Here we can link link to a different doc with in depth explanation of apps, libs and best practices: all the command line options, what should go into apps, how to structure libs, how to make sure lazy loading works. 

And here I would show how to build/test/serve apps and libs. 
```

With this in place, the workspace stuff should be covered, and we can talking about other things Nx provides.  Something like

```
Nx helps you have consistent code.

* Check out our schematics and runtime libraries for state management (Ngrx) and communicating with the server (link)
* Check out our schematics for upgrade (link)
* Check out our code formatter and lint checks (link)
...
```

What do you think?